package.json

```diff
@@ -39,6 +39,7 @@
     "gulp-sass-lint": "^1.1.1"
   },
   "dependencies": {
+    "eslint": "^2.11.1",
```

Comment:
sindresorhus/eslint-config-xo-space#3, needs to be ^2.3.0 to work but currently gulp-eslint requires only ^2.0.0.